### PR TITLE
Re-sync with uBlock master

### PR DESCRIPTION
**Pull request by ryanbr**

### CODE REVIEW

**Comment by ryanbr** [comment]
@brave/sec-team just syncing with upstream uBlock

File: assets/assets.json [hunk 1]
```diff
@@ -252,6 +233,13 @@
 		],
 		"supportURL": "https://forums.lanik.us/"
 	},
+	"ublock-annoyances": {
+		"content": "filters",
+		"group": "social",
```
Comment (fmarier) [inline assets/assets.json]: Do we use this `group` field in Brave? In other words, is this change significant?

Comment (ryanbr) [reply assets/assets.json]: I don't believe so, in brave://adblock is not ordered like uBo. Though its something we should copy (while offering other lists)

**Comment by ryanbr** [comment]
CLosing, will rebase this.

**Pull request closed without merging**